easybuild/easyconfigs/m/Mesa/Mesa-10.3.4-intel-2014b-Python-2.7.8.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'ConfigureMake'
+
+name = 'Mesa'
+version = '10.3.4'
+
+homepage = 'http://www.mesa3d.org/'
+description = """Mesa is an open-source implementation of the OpenGL specification -
+ a system for rendering interactive 3D graphics."""
+
+toolchain = {'name': 'intel', 'version': '2014b'}
+toolchainopts = {'optarch': True}
+
+sources = ['%(name)sLib-%(version)s.tar.gz']
+source_urls = [
+    'ftp://ftp.freedesktop.org/pub/mesa/%(version)s',
+    'ftp://ftp.freedesktop.org/pub/mesa/older-versions/%(version_major)s.x/%(version)s',
+]
+
+pythonver = '2.7.8'
+pythonshortver = '.'.join(pythonver.split('.')[0:2])
+versionsuffix = '-%s-%s' % ('Python', pythonver)
+
+builddependencies = [
+    ('flex', '2.5.39'),
+    ('Bison', '3.0.2'),
+    ('Autoconf', '2.69'),
+    ('Automake', '1.14.1'),
+    ('pkg-config', '0.28'),
+    ('libtool', '2.4.4'),
+    ('xproto', '7.0.26'),
+    ('xextproto', '7.3.0'),
+    ('glproto', '1.4.16'),
+    ('kbproto', '1.0.6'),
+]
+dependencies = [
+    ('Python', pythonver),
+    ('libxml2', '2.9.2', versionsuffix),
+    ('libdrm', '2.4.58'),
+    ('libX11', '1.6.2', versionsuffix),
+    ('libXext', '1.3.3', versionsuffix),
+    ('libXfixes', '5.0.1'),
+    ('libXdamage', '1.1.4', versionsuffix),
+    ('libXfont', '1.5.0', versionsuffix),
+    ('eudev', '3.0'),
```

Comment:
I suggest we do the same thing as we do with openssl: put an osdependency on udev and comment out the eudev easyconfig for those who really want it.